lib/menubar/init.lua

```diff
@@ -69,6 +69,8 @@ menubar.right_label = "▶▶"
 --- Label used for "Previous page", default "◀◀".
 menubar.left_label = "◀◀"
 
+local list_interspace = theme.xresources.apply_dpi(4) * 3 -- common.list_update padding
```

Comment:
Can you add a longer comment here instead? Something that explains that `common.list_update` adds three times a margin of `dpi(4)`?